src/Main.java

```diff
@@ -1,5 +1,60 @@
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+//Задачи 1, 2
+        int[] weight = {1, 2, 3};
+        double[] number = {1.57, 7.654, 9.986};
+        int[] apples = {5, 7, 56, 64, 23, 77};
+        for (int i = 0; i < weight.length; i = i + 1) {
+            if (i == 2) {
```

Comment:
нужно использовать длину массива, а не брать числа с потолка по сути
```suggestion
            if (i == weight.length - 1) {
```